.github/workflows/tests.yaml

```diff
@@ -39,10 +39,20 @@ jobs:
           key: ${{ runner.os }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
           restore-keys: |
             ${{ runner.os }}-pnpm-store-
-
+          
       - name: Install dependencies
         run: pnpm install --frozen-lockfile
-      
+
+      - name: Dry-run of build
+        shell: bash
+        run: |
```

Comment:
I think there's a cli arg that specifies a dry run here. Also curious why the pipe operator?

Reply:
@DrewHoo I looked for dry run, but the only thing I could find was `tsc --build --dry` and that doesn't fail like `tsc -p` does 🤷🏻

Reply:
Did you look at https://semantic-release.gitbook.io/semantic-release/usage/configuration#dryrun

If this is complicated, i think it's fine to defer & tackle later in another PR

Reply:
I tried `npx semantic-release --dry-run`, but it doesn't fail that way either.